pkg/resources/test/load_files_test.go

```diff
@@ -294,7 +294,8 @@ func TestLoadFiles(t *testing.T) {
 							Name: "de-test-01",
 							UID:  types.UID("1234567890"),
 							Labels: map[string]string{
-								"my-label": "my-value",
+								"my-label":                     "my-value",
+								kubermaticv1.ProjectIDLabelKey: "project",
```

Comment:
I suggest this change to not confuse it with kind
```suggestion
								kubermaticv1.ProjectIDLabelKey: "my-project",
```